### DISCUSSION TITLE
Weird formatting with spells

### DISCUSSION THREAD

**Discussion by weaverbosity** [Obsidian-TTRPG-Community/fantasy-statblocks#266] [Q&A]
I'm not sure how I've made this happen, but I have.

When I make a list of spells in a homebrew statblock (i.e. pull in a character from Dragonheist), I get "statblock-link" all over the stat block.

![Ammalia_Statblock](https://github.com/javalent/fantasy-statblocks/assets/127628234/cb78e19a-c3f4-4e44-91a7-ac0f598a9610)

Here's what the edit screen looks like.

![Ammalia_Edit Monster](https://github.com/javalent/fantasy-statblocks/assets/127628234/e4a79f4e-93d8-48df-a0b5-ababcfa3df12)

Am I doing something wrong here?

**Comment by valentine195** [comment]
Can you open the developer console and look for errors? ctrl/shift/i 

the plugin transforms links into that and then transforms them back into links later to avoid issues with YAML, so something is preventing it from reaching that transform.

**Comment by weaverbosity** [comment]
Could this be it?

![image](https://github.com/javalent/fantasy-statblocks/assets/127628234/b3d4f8fa-a36d-48b1-892a-b9c2839d3a8c)
